run.sh

```diff
@@ -294,6 +302,26 @@ workload_1_java() {
                                 -warmup="${WARMUP}"
 }
 
+workload_bidi_java() {
```

Comment:
Do these need different names? Not sure how bidi is turned on in the cli but shouldn't a workload 1 (or whatever workload applicable) with a`-test_type` set to "bidi" be enough?

Reply:
The bidi workload is completely different from the W1R3 workload.

Reply:
also i didn't think bash had overloading on functions so by default we need different workloads for different test_types since they aren't a parameter?

Reply:
> The bidi workload is completely different from the W1R3 workload.

What workload is this?

Reply:
it just runs differently. bidi write is only exposed via blobWriteSession directly so for bidi metrics we are just performing a bidi write and publishing the metrc. whereas our W1R3 use createFrom and downloadTo. CreateFrom does not use blobWriteSession to write an object.

Reply:
Recommend naming this "workload_bidi" / "nobidi" workload by a different name since it's not existing workload type and bidi be a flag that can be enabled.

Reply:
I am not sure if there is a better fit. workload write session? it really only makes sense under the context of bidi. The reason we even need 2 is because we need to be able to compare bidi to nonbidi and we cannot publish 2 sets of metrics under the same workload.

Reply:
the workload name for metric comes from the name of the prober in cloudprober cfg so that can stay the same;  

maybe: `workload_write_only` and flag `bidi_enabled`.

Reply:
this flag is to live in the cfg? bidi_enabled vs disabled and the run script is to pipe this through?

Reply:
Yes, you would create a prober for bidi_enabled vs. enabled and supply the bidi_enabled flag only for bidi prober in cfg.

Reply:
sorry about the delay. should look better now

Reply:
depends on https://github.com/googleapis/java-storage/pull/2468

Reply:
I agree with Frank here. `bidi` can be a flag passed through the cfg that gets captured by the script, and the workload would be `write_only` (or we could give it a number). That way, the workload can be tested with /without bidi, and with  other options if necessary in the future